### ISSUE TITLE
How to draw and fill ellipses?

### ISSUE THREAD

**Issue by J-F-Liu** [jrmuizel/raqote#120]
Including oblique ellipses.

**Comment by jrmuizel** [comment]
There's some code to create an ellipse here: https://github.com/servo/servo/blob/master/components/canvas/raqote_backend.rs#L701

Does that do what you want?

**Comment by J-F-Liu** [comment]
Yes, it uses `lyon_geom::Arc` to create quadratic beziers. Better include something similar to raqote.
